qa/tasks/ceph.py

```diff
@@ -433,6 +433,11 @@ def cephfs_setup(ctx, config):
         if max_mds > 1:
             fs.set_max_mds(max_mds)
 
+        # If absent will use the default value (60 seconds)
+        session_timeout = config.get('sesstion_timeout', 60)
+        if session_timeout != 60:
+            fs.set_session_timeout(session_timeout)
```

Comment:
```suggestion
        session_timeout = config.get('sesstion_timeout')
        if session_timeout is not None:
            fs.set_session_timeout(session_timeout)
```

(and move this to `cephfs/filesystem.py`)

Reply:
I didn't get it here. In `filesystem.py` I added one `set_session_timeout()` method. 
Do you mean move all the 3 lines to `filesystem.py` ?

Reply:
I mean the session tiemout should be configured in Filesystem.create if the override is set.